### PR TITLE
Fix doctring for preferences tuple in dle.py

### LINKED ISSUE #617: Fix `DLE` docstring
Docstring for `DLE` (the order of  the elements of `tuple`) https://github.com/QuantEcon/QuantEcon.py/blob/496fdeaa22460824bba2ce58a7fb908e3f2216b7/quantecon/dle.py#L55-L57 does not match the code https://github.com/QuantEcon/QuantEcon.py/blob/496fdeaa22460824bba2ce58a7fb908e3f2216b7/quantecon/dle.py#L66

### PR DESCRIPTION
**Pull request by faze-geek**
Fix #617

### CODE REVIEW

**Comment by faze-geek** [comment]
@oyamad Could you have a look ?

File: quantecon/dle.py [hunk 1]
```diff
@@ -53,8 +53,8 @@ class DLE(object):
         Technology is a tuple containing the matrices \Phi_c, \Phi_g, \Phi_i, \Gamma,
         \Delta_k, and \Theta_k
     Preferences : tuple
-        Preferences is a tuple containing the matrices \Delta_h, \Theta_h, \Lambda,
-        \Pi, and the scalar \beta
+        Preferences is a tuple containing the matrices scalar \beta, \Lambda,
+        \Pi, \Delta_h, and \Theta_h
```
Comment (oyamad) [inline quantecon/dle.py:57]: @faze-geek Thank you, but mind the "matrices":

```suggestion
        Preferences is a tuple containing the scalar \beta and the
        matrices \Lambda, \Pi, \Delta_h, and \Theta_h
```

Comment (faze-geek) [reply quantecon/dle.py]: I have made the suggested change . Would also appreciate if you could approve running workflows.

**Review by oyamad — APPROVED** [review]

**Comment by oyamad** [comment]
@faze-geek Thanks! Merging.

**Pull request merged**